### PR TITLE
update go 1.19.1

### PR DESCRIPTION
**Pull request by to-lz1**
* updated Go version to v1.19.1 which includes security update.
* removed io/ioutil which has been deprecated in Go 1.16.

CI/CD example(including tag-release):
https://github.com/to-lz1/dsps/actions

### CODE REVIEW

**Comment by to-lz1** [comment]
@hirosassa Please review for Go update of Dsps 🙏

**Comment by hirosassa** [comment]
@to-lz1 Thanks for your contribution! 

I wonder why the CIs didn't run 🤔 . I'll take a look.

File: .github/workflows/client-js-test.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
 
     runs-on: ubuntu-latest
     env:
-      go-version: 1.18.3
+      go-version: 1.19.1
 
     steps:
       - uses: actions/checkout@v2
```
Comment (hirosassa) [inline .github/workflows/client-js-test.yaml]:
```suggestion
      - uses: actions/checkout@v3
```

Comment (hirosassa) [reply .github/workflows/client-js-test.yaml]: This line should also be v3, too.
https://github.com/m3dev/dsps/pull/8/files#diff-35e16cd18acb537bf3ce35e5b3eb25723dc3013dc5d2cab1e400a7de8066f8f7R35

**Review by hirosassa — CHANGES_REQUESTED** [review]

**Pull request merged**